litellm/proxy/_experimental/mcp_server/mcp_server_manager.py

```diff
@@ -78,6 +78,8 @@ def load_servers_from_config(self, mcp_servers_config: Dict[str, Any]):
         """
         verbose_logger.debug("Loading MCP Servers from config-----")
         for server_name, server_config in mcp_servers_config.items():
+            if '-' in server_name:
```

Comment:
please use one helper util called validate MCP server name, if it includes "-" then raise exception

Reply:
Done